sdk/keyvault/Azure.Management.KeyVault/CHANGELOG.md

```diff
@@ -2,5 +2,75 @@
 
 ## 1.0.0-preview.1
 
-### New Features
-- Initial preview of Azure Management KeyVault SDK based on Azure.Core
\ No newline at end of file
+This is public preview version based on Azure Core. This version uses a next-generation code generator that introduces important new features.
+
+### General New Features
+
+    - Support MSAL.NET, Azure.Identity is out of box for supporting MSAL.NET
+    - Support [OpenTelemetry](https://opentelemetry.io/) for distributed tracing
```

Comment:
Add

-HTTP Pipeline with custom policies
-Error-handling that matches language ecosystem